pkgs/development/python-modules/courlan/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, langcodes
+, pytest
+, tld
+, urllib3
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "courlan";
+  version = "0.9.3";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-fm3Sp+V4U7lNuWVu4XabjNrMu0mCNGiRxCM4Mp1+CVg=";
```

For SRi hashes please use `hash`. 

```suggestion
    hash = "sha256-fm3Sp+V4U7lNuWVu4XabjNrMu0mCNGiRxCM4Mp1+CVg=";
```